### PR TITLE
Promote Nicky/GEHACK autologin

### PR DESCRIPTION
**Pull request by vmcj**

### CODE REVIEW

File: website/tools.shtml [hunk 1]
```diff
@@ -62,4 +62,11 @@
     can connect to.
 </p>
 
+<h3>Autologin for contestantmachine</h3>
+
+<p> <a href="https://github.com/GEHACK/lightdm-qt5-greeter-ccs-api">LightDM CCS autologin</a> makes it possible
+    to autologin contestantmachines when the contest starts in DOMjudge. This is useful to keep problemdata secret
```
Comment (tuupke) [inline website/tools.shtml:68]: isn't to "automatically log into contestant machines" better? Maybe replace contestant with "team". Though I'm fine with this too

**Review by tuupke — APPROVED** [review]
Maybe one thing to change, but lgtm

File: website/tools.shtml (hunk 1 above)
Comment (eldering) [reply website/tools.shtml]: "contestant machine" should be written separately (in general in English you rarely combine words).

File: website/tools.shtml [hunk 2]
```diff
@@ -62,4 +62,11 @@
     can connect to.
 </p>
 
+<h3>Autologin for contestantmachine</h3>
```
Comment (eldering) [inline website/tools.shtml:65]:
```suggestion
<h3>Autologin for team machines</h3>
```

File: website/tools.shtml [hunk 3]
```diff
@@ -62,4 +62,11 @@
...
+
+<p> <a href="https://github.com/GEHACK/lightdm-qt5-greeter-ccs-api">LightDM CCS autologin</a> makes
+    it possible to automatically log into team machines when the contest starts in DOMjudge. This
+    is useful to keep problemdata secret and to make sharing credentials easier on the organizer.
```
Comment (eldering) [inline website/tools.shtml:69]:
```suggestion
    it possible to automatically log in to team machines when the contest starts in DOMjudge. This
    is useful to keep problem data secret and to make sharing credentials easier on the organizer.
```

**Pull request merged**